plugin.video.watchmojo/addon.py

```diff
@@ -17,156 +17,12 @@
-    for elem in menu_elem.iterfind(".//a"):
-        url = elem.get("href")
-        if url and elem.text and elem.text != "MsMojo":
-            item = Listitem()
-            item.label = elem.text
-            item.set_callback(video_list, url=url)
-            yield item
-
-
-@Route.register
-def video_list(plugin, url):
-    """
-    List all video for given url.
-
-    site: http://www.watchmojo.com/shows/Top%2010
-
-    :param Route plugin: Tools related to Route callbacks.
-    :param unicode url: The url to a list of videos.
-    :return: A generator of listitems.
-    """
-    url = url_constructor(url)
-    source = urlquick.get(url)
-    lbl_tags = plugin.localize(TAGS)
-
-    # Parse all the video elements
-    root_elem = source.parse()
-
-    for line in root_elem.iterfind(".//div[@class='owl-carousel margin-bottom']"):
-        for elem in line.iterfind(".//div[@class='item']"):
-            yield extract_videos(lbl_tags, elem, "%b %d, %Y")
-
-    # Add link to next page if available
-    next_page = root_elem.find(".//div[@class='cat-next']")
-    if next_page is not None:
-        url = next_page.find("a").get("href")
-        yield Listitem.next_page(url=url)
-
-
-@Route.register
-def related(plugin, url):
-    """
-    List all related videos to selected video.
-
-    site: http://www.watchmojo.com/video/id/19268/
-
-    :param Route plugin: Tools related to Route callbacks.
-    :param unicode url: The url to a video.
-    :return: A generator of listitems.
-    """
-    url = url_constructor(url)
-    source = urlquick.get(url)
-    lbl_tags = plugin.localize(TAGS)
-
-    # Parse all the video elements
-    root_elem = source.parse("div", attrs={"id": "owl-demo1"})
-    for elem in root_elem.iterfind(".//div[@class='item']"):
-        yield extract_videos(lbl_tags, elem, "%B %d, %Y")
-
-
-@Route.register
-def tags(plugin, url):
-    """
-    List tags for a video.
-
-    site: https://www.watchmojo.com/video/id/19268/
-
-    :param Route plugin: Tools related to Route callbacks.
-    :param unicode url: The url to a video.
-    :return: A generator of listitems.
-    """
-    plugin.category = plugin.localize(TAGS)
-    url = url_constructor(url)
-    source = urlquick.get(url)
-
-    # Parse all video tags
-    root_elem = source.parse("div", attrs={"id": "tags"})
-    for elem in root_elem.iterfind("a"):
-        item = Listitem()
-        item.label = elem.text.title()
-        item.set_callback(video_list, url="%s1" % elem.get("href"))
-        yield item
-
-
-@Resolver.register
-def play_video(plugin, url):
-    """
-    Resolve video url.
-
-    site: https://www.watchmojo.com/video/id/19268/
-
-    :param Resolver plugin: Tools related to Resolver callbacks.
-    :param unicode url: The url to a video.
-    :return: A playable video url.
-    """
-    url = url_constructor(url)
-    return plugin.extract_source(url)
+import sys
+import os
 
+lib = os.path.join(os.path.dirname(__file__), "resources", "lib")
```

Comment:
please properly turn this into a python module by creating `__init__` files along the paths you need to import. `sys.path.append` is not a clean way to import your dependencies.